src/components/Page/index.tsx

```diff
@@ -76,7 +78,7 @@ export default function Page({
       <Grid item xs={12}>
         <PageSnackbar />
       </Grid>
-      <Grid container ref={contentRef} sx={contentStyle}>
+      <Grid container ref={pageHeaderRef ?? contentRef} sx={contentStyle}>
```

Comment:
I think I'm missing when/why we would want to override the ref.

Reply:
The table container uses the page header ref. I'm not exactly sure what it is for, but since the table component is adjacent to this one, it has to be managed in the parent. I can double check tomorrow to see if it is needed.

Reply:
The page header ref is mostly to manage the static header when scrolling. Page component should auto handle it I think. Sure, if you figure it out and it's not needed, feel free to remove it. Looks great otherwise.